docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -2,51 +2,109 @@
+in this script: [commands/push-blueprint.sh](./commands/push-blueprint.sh).
+
+
+## Components and Component Descriptors
+
+An Installation may reference its blueprints via so-called
+[component-descriptors](../../../concepts/Glossary.md#_component-descriptor_).  A component descriptor describes a
+component, or rather, a specific component version. In general, a component version is a container for all required
+resources for the deployment of a specific version of an application or software system. In this example, the
+application is the hello-world application deployable with the landscaper through an external blueprint. Thereby, the
+external blueprint is the only resource required for the deployment.  A component version may either contain a resource
+through referencing it at an external location (such as an oci registry) or through embedding it as a local blob.
+
+#### Component Version with External Resource
+
+A file system representation of a component version containing the resource through an external reference is shown
+[here](./component-archive/v2-external).  The corresponding component-descriptor describing that component version is
+stored as a top-level file. The component descriptor contains only a single _resource_ with the `name: blueprint`. This
+resource has an _access_ of `type: ociArtifact` that contains a reference to the previously uploaded image.  Besides the
+component-descriptor, there is a directory called blobs at the top-level. This is where the local blobs of a embedded
+resource would be located. Since the blueprint is the only resource of this component version and it is contained
+through an external reference, this directory is empty.
+
+#### Component Version with Local Resource 
+
+A file system representation of a component version containing the resource as a local blob is shown
+[here](./component-archive/v2-local).  Again, the corresponding component-descriptor describing that component version
+is stored as a top-level file. Exactly as before, the component descriptor contains only a single _resource_ with the
+`name: blueprint`. But now, this resource has an _access_ of `type: localBlob`. Instead of an `imageReference`, this
+_access_ of `type: localBlob` has a `localReference`. This is the sha256 hash value of the blueprint. If you open the
+blob directory here, you will see that it contains a file with exactly that name. Furthermore, this _access_ has a field
+`mediaType`, which provides information about the format in which the blueprint is stored, here that it is archived
+and compressed (tar+gzip). The _access_ of `type: ociArtifact` did not need to provide this information since the
+`ociArtifact` format is determined through the oci standard and the format of the contents of the oci artifact is
+described within the artifact itself.  
+  
+These file system representations of component versions can then be uploaded to an oci registry.  
+
```

Comment:
I'm not sure if we should move this section to the end because it interrupts the current example.

Reply:
Yeah, you are probably right. I worried that this might interrupt the tutorial flow too much while writing it as well. I left it there, because I thought it is kind of essential for a basic understanding of the ocm.

I restructured it now and moved the largest part to the end of the section.